### ISSUE TITLE
Show hashing progress

### ISSUE THREAD

**Issue by CyberShadow** [feross/webtorrent-cli#32]
When resuming a download of a large torrent, there is no progress indication for rehashing already-downloaded content. This is the only output:

```
(node:17893) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 ready listeners added. Use emitter.setMaxListeners() to increase limit
```

**Comment by CyberShadow** [comment]
There is some existing code for this, `torrent.on('verifying', ...)`, however the `verifying` event is not listed on https://webtorrent.io/docs.

**Comment by CyberShadow** [comment]
I don't see this event being implemented in the webtorrent library. I'm guessing it would need to be implemented in `_verfyPieces` or `_markVerified` there.

**Comment by feross** [comment]
Thanks – fixed in 1.9.2
